.github/mergify.yml

```diff
@@ -0,0 +1,61 @@
+queue_rules:
+  - name: default
+    conditions:
+      - "#approved-reviews-by>=1"
+      - base=main
+      - check-success=📚 Documentation
+      - check-success=Schutzbot on GitLab
+      - check-success=Test Suite (test.mod)
+      - check-success=Test Suite (test.run.test_assemblers)
+      - check-success=Test Suite (test.run.test_boot)
+      - check-success=Test Suite (test.run.test_noop)
+      - check-success=Test Suite (test.run.test_sources)
+      - check-success=Test Suite (test.run.test_stages)
+      - check-success=Test Suite (test.src)
+      - check-success=Regenerate Test Data
+      - check-success=Spell check
+      - "check-success=LGTM analysis: Python"
+      - check-success=codecov/project
+      - "check-success~=rpm-build:.*"
+    
+pull_request_rules:
+  - name: Automatic merge for Dependabot pull requests
+    conditions:
+      - author~=^dependabot(|-preview)\[bot\]$
+      - title~=^Bump [^\s]+ from ([\d]+)\..+ to \1\.
+      - base=main
+      - check-success=📚 Documentation
+      - check-success=Schutzbot on GitLab
+      - check-success=Test Suite (test.mod)
+      - check-success=Test Suite (test.run.test_assemblers)
+      - check-success=Test Suite (test.run.test_boot)
+      - check-success=Test Suite (test.run.test_noop)
+      - check-success=Test Suite (test.run.test_sources)
+      - check-success=Test Suite (test.run.test_stages)
+      - check-success=Test Suite (test.src)
+      - check-success=Regenerate Test Data
+      - check-success=Spell check
+      - "check-success=LGTM analysis: Python"
+      - check-success=codecov/project
+      - "check-success~=rpm-build:.*"
+    actions:
+      queue:
+        name: default
+        method: rebase
```

Comment:
let's also maybe set `rebase_fallback` to `none`?

Reply:
good idea!